components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/LoginEventHookHandler.java

```diff
@@ -0,0 +1,230 @@
+import org.wso2.identity.webhook.common.event.handler.model.ResourceConfig;
+import org.wso2.identity.webhook.common.event.handler.util.EventHookHandlerUtils;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.base.IdentityRuntimeException;
+import org.wso2.carbon.identity.configuration.mgt.core.exception.ConfigurationManagementException;
+import org.wso2.carbon.identity.configuration.mgt.core.model.Attribute;
+import org.wso2.carbon.identity.configuration.mgt.core.model.Resources;
+import org.wso2.carbon.identity.configuration.mgt.core.search.ComplexCondition;
+import org.wso2.carbon.identity.configuration.mgt.core.search.Condition;
+import org.wso2.carbon.identity.configuration.mgt.core.search.PrimitiveCondition;
+import org.wso2.carbon.identity.configuration.mgt.core.search.constant.ConditionType;
+import org.wso2.carbon.identity.core.bean.context.MessageContext;
+import org.wso2.carbon.identity.event.IdentityEventConstants;
+import org.wso2.carbon.identity.event.IdentityEventException;
+import org.wso2.carbon.identity.event.bean.IdentityEventMessageContext;
+import org.wso2.carbon.identity.event.event.Event;
+import org.wso2.carbon.identity.event.handler.AbstractEventHandler;
+import org.wso2.carbon.utils.multitenancy.MultitenantConstants;
+import org.wso2.identity.event.common.publisher.model.EventPayload;
+import org.wso2.identity.event.common.publisher.model.SecurityEventTokenPayload;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.wso2.carbon.identity.configuration.mgt.core.search.constant.ConditionType.PrimitiveOperator.EQUALS;
+import static org.wso2.identity.webhook.common.event.handler.constant.Constants.EVENT_SCHEMA_TYPE_WSO2;
+
+/**
+ * Login Event Hook Handler.
+ */
+public class LoginEventHookHandler extends AbstractEventHandler {
+
+    private static final Log log = LogFactory.getLog(LoginEventHookHandler.class);
+    private final EventHookHandlerUtils eventHookHandlerUtils;
+    private final EventConfigManager eventConfigManager;
+
+    public LoginEventHookHandler(EventHookHandlerUtils eventHookHandlerUtils,  EventConfigManager eventConfigManager) {
+        this.eventHookHandlerUtils = eventHookHandlerUtils;
+        this.eventConfigManager = eventConfigManager;
+    }
+
+    @Override
+    public String getName() {
+
+        return Constants.LOGIN_EVENT_HOOK_NAME;
+    }
+
+    @Override
+    public boolean canHandle(MessageContext messageContext) throws IdentityRuntimeException {
+
+        IdentityEventMessageContext identityContext = (IdentityEventMessageContext) messageContext;
+        String eventName = identityContext.getEvent().getEventName();
+
+        boolean canHandle = isSupportedEvent(eventName);
+        log.debug("canHandle() returning " + canHandle + " for the event: " + eventName);
```

Comment:
Better just say "eventName can be handled"

Reply:
Addressed with https://github.com/wso2-extensions/identity-webhook-event-handlers/pull/1/commits/dc72213e0cb8089d982fddd20de1d085f01ac28a